frontend/src/utils/user-agent.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * A user agent string by which Openverse.org identifies itself to services
+ */
+const userAgent =
+  "Openverse/0.1 (https://openverse.org; openverse@wordpress.org)"
```

Comment:
Nit, potentially non-relevant: this version is rather arbitrary.

Reply:
Agreed, I copied this from our usage somewhere else, but now I don't remember.  I'll try to find it and reassess an alternative

Reply:
Nit: should this file go to the `/constants` folder instead?

Reply:
We use `0.1` for the catalog as well it seems: 

https://github.com/WordPress/openverse/blob/e1f48221938b4cc9838a85578b47bb4f6bfa33f6/catalog/dags/common/loader/provider_details.py#L121